schema/vrs.json

```diff
@@ -223,7 +223,60 @@
                ]
             },
             "copies": {
-               "$ref": "#/definitions/AbsoluteCopyCount"
+               "oneOf": [
```

Comment:
This will be easier to use if it's always a Range, possibly with equal endpoints.